### PR TITLE
Octan Network

### PR DESCRIPTION
**Pull request by quangtran8588**
#### Project Abstract

> Please replace these instructions with a brief description of your project summarising key points (1-2 paragraphs).
>
> If your application is a follow-up to a previous grant, please mention which one in the first line of the abstract and include a link to previous pull requests if applicable.

#### Grant [level](https://github.com/w3f/Grants-Program#level_slider-levels)

- [ ] **Level 1**:  Up to $10,000, 2 approvals
- [x] **Level 2**:  Up to $30,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 5 approvals (for >$100k: Web3 Foundation Council approval)

#### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied and aptly renamed (`project_name.md`).
- [x] I have read the [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/grant_guidelines_per_category.md).
- [x] Payment details have been provided (bank details via email _or_ Polkadot (USDC & USDT) or BTC address in the application).
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process) (see the [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/announcement-guidelines.md)).
- [ ] I prefer the discussion of this application to take place in a private Element/Matrix channel. My username is: `@_______:matrix.org` (change the homeserver if you use a different one)

### CODE REVIEW

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks for your interest in our grants program. I have some initial feedback/questions: The default [deliverables of the template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) must be added to each milestone. Furthermore, we don't support the deployment via our grants program. Also, how is your indexing solution different from already existing open-source tools? See: https://wiki.polkadot.network/docs/build-open-source

**Comment by paven86** [comment]
> **CLA Assistant Lite bot:** Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/T%26Cs.md) before we can accept your contribution. Please submit the following text as a separate comment:
> 
> I have read and hereby sign the Contributor License Agreement.
> 
> **1** out of **2** committers have signed the CLA.✅ @quangtran8588❌ @paven86You can retrigger this bot by commenting **recheck** in this Pull Request

I have read and hereby sign the Contributor License Agreement.

**Comment by paven86** [comment]
I have read and hereby sign the Contributor License Agreement.

**Comment by paven86** [comment]
> Thanks for your interest in our grants program. I have some initial feedback/questions: The default [deliverables of the template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) must be added to each milestone. Furthermore, we don't support the deployment via our grants program. Also, how is your indexing solution different from already existing open-source tools? See: https://wiki.polkadot.network/docs/build-open-source

Thanks for your feedbacks. We have updated the default deliverables. Additionally, Octan Network doesn't re-invent indexing techniques. _Octan's KEY INNOVATIONs_ are **onchain Reputation Ranking System** (powered by adaptive PageRank algorithms, our 5-year research) to compute reputation scores from onchain TXNs, and a **Soulbound** standard to carry reputation scores onchain. These are to support Web3 entities building their onchain trustworthiness without compromising privacy. Moreover, Octan Reputation Ranking & Analytics extract social insights, social inter-relationships of onchain entities, measure individual persona, user behaviors & segmentation, which are all useful for Web3 marketing profession (e.g. qualifying & classifying users), for market research (e.g. investigating various categories, importance & impact of dapps within Web3 ecosystems, etc). Therefore, Octan Network is distinguished from all other onchain data analytics firms (e.g. Nansen, Chainalysis, Dune Analytics, Arkham, Web3Go) who only focus on tracing & tracking cash-flows, analyzing financial indicators & features.

File: applications/Octan_Network.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
+# Octan Network
+
+> This document will be part of the terms and conditions of your agreement and therefore needs to contain all the required information about the project. Don't remove any of the mandatory parts presented in bold letters or as headlines (except for the title)! Lines starting with a `>` (such as this one) should be removed. Please use markdown instead of HTML (e.g. `![](image.png)` instead of `<img>`). 
+>
+> See the [Grants Program Process](https://github.com/w3f/Grants-Program/#pencil-process) on how to submit a proposal.
+- **Team Name:** Octan Labs
+- **Payment Address:** Ethereum address (DAI) 0xC0e09A112Ae45d87597CD78c11b7D95a55aCC5F0
```
Comment (semuelle) [inline applications/Octan_Network.md:7]: We don't offer ERC20 payments anymore. Please refer to the readme and application template for payment options.

File: applications/Octan_Network.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+- Long Nguyen: senior business growth
+### Contact
+
+- **Contact Name:** Paven Do
+- **Contact Email:** paven@octan.network
+- **Website:** [https://octan.network/](https://octan.network/)
+
+### Legal Structure
+
+- **Registered Address:** unregistered
```
Comment (semuelle) [inline applications/Octan_Network.md:74]: Please provide an address, either here or via [email](mailto:grants@web3.foundation).

File: applications/Octan_Network.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+| **0a.** | License | Unlicense |
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains the onchain reputation concept and how a user can use the reputation board. |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Article | We will publish an **article**/workshop that explains the concept of onchain reputation and the vision of Octan Network. |
+| 1a. | Crawling raw data | We will build module for indexing onchain transactions, extrinsics, events from archive node |
+| 1b. | Data Warehouse slot | We will setup data warehouse slot and structure for query onchain performance related metrics |
+| 2a. | Reputation ranking algorithms | We will complete and publish a (scientific) research article about onchain reputation ranking algorithms on blockchain-based transaction datasets. |
```
Comment (semuelle) [inline applications/Octan_Network.md:144]: Why is "scientific" in brackets? What does "complete" mean? Published in a peer-reviewed medium? I advise against that, as you probably know peer review can take a very long time.

File: applications/Octan_Network.md [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+- *Reputation Board:* an interface for users to search onchain addresses (and/or dapps), their reputation scores & ranking, onchain statistics.
+- *Analytics platform:* an interface for customers to query reputation scores, onchain statistics, visualize and analyze them, to extract social insights and segment users. The platform also provides an interface to label onchain entities, classify contracts and dapps into different categories (e.g. DeFi, NFT, DAO, socialfi, gamefi, etc), making onchain entities much easier to understand and analyze.
+- *1ID & Soulbound:* an wallet management tool, allowing users to mint soulbound token carrying their reputation score as a proof of trust and contribution onchain. Octan 1ID also offers interface for users to track their onchain activities.
+
+Current **Architecture Overview** of Octan Network
+
+![Current Architecture Overview of Octan Network](https://github.com/Octan-Labs/Grants-Program/assets/45308207/9cade4e3-989f-485f-8ce5-5cd82611b1a9)
```
Comment (semuelle) [inline applications/Octan_Network.md:31]: What does `analyze to earn` refer to here?

**Review by semuelle — CHANGES_REQUESTED** [review]
Thanks for the application, @quangtran8588. A couple of questions and requests:

- Please provide a tech stack for all deliverables.
- Please only list deliverables that are reproducible. "Set up data warehouse slot" doesn't fit the scope of the grants program (see FAQ and guidelines).
- Your competitor analysis only mentions Web3Go, which doesn't do reputation at all. Could you find comparable projects in other ecosystems that could give us an idea of the size of the target audience?
- You call it an "on-chain reputation system", but it's currently not clear to me which parts are actually on-chain. Could you clarify which deliverables are actually on-chain, how they are implemented and what data they hold?

**Comment by trmaphi** [comment]
I have read and hereby sign the Contributor License Agreement.

**Comment by trmaphi** [comment]
I have read and hereby sign the Contributor License Agreement.

File: applications/Octan_Network.md (hunk 1 above)
Comment (paven86) [reply applications/Octan_Network.md]: Dear Semuelle, we committed new changes:

- Provide tech stacks for deliverables
- Add competitors about onchain reputation concept
- By Octan, "onchain reputation" means: (i) reputation scores are computed from onchain transactions; (ii) Web3 users mint Soulbound tokens to update and prove their reputation scores onchain.
- Changed ERC20 to Polkadot address
- Provided physical address
- Removed "scientific" attribute from research paper
- Removed "label to earn"

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks again for the application and all the effort you put into this. However, the grants committee decided today not to go ahead with the application. We generally don't support the development of solidity smart contracts and have already supported quite a lot of different indexing solutions. Independent of it, we wish you all the best for your project.

**Pull request closed without merging**